plugin/src/main/java/org/jetbrains/research/refactoringminer/OutputRefactoringHandler.java

```diff
@@ -0,0 +1,46 @@
+package org.jetbrains.research.refactoringminer;
+
+import org.jetbrains.annotations.NotNull;
+import org.refactoringminer.api.Refactoring;
+import org.refactoringminer.api.RefactoringHandler;
+import org.refactoringminer.rm1.GitHistoryRefactoringMinerImpl;
+import java.io.IOException;
+import java.util.List;
+
+public class OutputRefactoringHandler extends RefactoringHandler {
+    private final boolean verbose;
```

Comment:
what is the purpose of this variable?

Reply:
Whether to print the commit statistics at the end of processing